diff --git a/library/general/src/modules/Arch.rb b/library/general/src/modules/Arch.rb
index a52aab4f..8d9dcb40 100644
--- a/library/general/src/modules/Arch.rb
+++ b/library/general/src/modules/Arch.rb
@@ -516,6 +516,18 @@ module Yast
       SCR.Read(path(".target.string"), "/proc/sys/kernel/osrelease").to_s.downcase.include?("-microsoft")
     end
 
+    # Check whether running in a management container. The target system location
+    # is then specified in the Installation.destdir value.
+    #
+    # @note We need to distinguish between running in a container and managing
+    # the container itself or managing the host system which runs the container.
+    #
+    # @return [Boolean] true if runs in a management container
+    def is_management_container
+      target_dir = ENV["YAST_TARGET_DIR"] || ""
+      File.directory?(target_dir)
+    end
+
     # map the Arch.architecture to the arch expected by SCC
     RPM_ARCH = {
       "s390_32" => "s390",
