diff --git a/library/control/src/modules/Installation.rb b/library/control/src/modules/Installation.rb
index 94b132a2..aba82c45 100644
--- a/library/control/src/modules/Installation.rb
+++ b/library/control/src/modules/Installation.rb
@@ -212,6 +212,9 @@ module Yast
       elsif Stage.initial
         @destdir = "/mnt"
         @scr_destdir = "/mnt"
+      elsif WFM.scr_chrooted?
+        @destdir = WFM.scr_root
+        @scr_destdir = WFM.scr_root
       end
 
       nil
