diff --git a/library/control/src/modules/Installation.rb b/library/control/src/modules/Installation.rb
index 94b132a2..9951046b 100644
--- a/library/control/src/modules/Installation.rb
+++ b/library/control/src/modules/Installation.rb
@@ -214,6 +214,15 @@ module Yast
         @scr_destdir = "/mnt"
       end
 
+      target_dir = ENV["YAST_TARGET_DIR"] || ""
+      if !target_dir.empty?
+        if File.directory?(target_dir)
+          @destdir = target_dir
+        else
+          abort "Cannot set the target, directory #{target_dir} not found"
+        end
+      end
+
       nil
     end
 
