diff --git a/src/modules/Sysconfig.rb b/src/modules/Sysconfig.rb
index 401bd6b..b5b24b1 100644
--- a/src/modules/Sysconfig.rb
+++ b/src/modules/Sysconfig.rb
@@ -32,13 +32,11 @@ module Yast
 
 
       @configfiles = [
-        "/etc/sysconfig/*",
-        "/etc/sysconfig/network/ifcfg-*",
-        "/etc/sysconfig/network/dhcp",
-        "/etc/sysconfig/network/config",
+        "/mnt/etc/sysconfig/*",
+        "/mnt/etc/sysconfig/network/ifcfg-*",
+        "/mnt/etc/sysconfig/network/dhcp",
+        "/mnt/etc/sysconfig/network/config",
         Ops.add(Directory.ydatadir, "/descriptions"),
-        "/etc/sysconfig/powersave/*",
-        "/etc/sysconfig/uml/*"
       ]
 
       # Additional files from Import
@@ -1152,11 +1150,16 @@ module Yast
         log.warn("No absolute path in command #{cmd}") unless cmd.lstrip.start_with?("/")
 
         log.info "Starting: #{cmd}"
-        cmd_out = SCR.Execute(path(".target.bash_output"), "#{cmd} 2>&1")
-        log.info "Result: #{cmd_out['exit']}"
-        log.info "Output: #{cmd_out['stdout']}"
-
-        cmd_out["exit"] == 0
+        #  cmd_out = SCR.Execute(path(".target.bash_output"), "#{cmd} 2>&1")
+        begin
+          cmd_out = `chroot /mnt #{cmd} 2>&1`
+          log.info "Result: #{$?.exitstatus}"
+          log.info "Output: #{cmd_out}"
+
+          $?.exitstatus.zero?
+        rescue
+          false
+        end
       end
 
       exec_action(action, label, error, confirm)
